app/models/user.rb

```diff
@@ -1,5 +1,5 @@
 class User < ApplicationRecord
-  validates :name, :email, :password_digest, presence: true
+  validates :name, :email, presence: true
```

Comment:
Why are we removing this validation? Are we still going to be storing password_digest? Or is it just that we don't need to validate?